transport/src/main/java/io/netty/channel/ChannelPipeline.java

```diff
@@ -344,6 +417,79 @@
      */
     ChannelPipeline addAfter(EventExecutorGroup group, String baseName, String name, ChannelHandler handler);
 
+    /**
+     * Inserts a {@link ChannelHandler} after the specified {@link ChannelHandler} of this pipeline.
+     *
+     * @param  baseHandler   the existing {@link ChannelHandler}
+     * @param  newHandler    the {@link ChannelHandler} to insert after
+     *
+     * @return itself
+
+     * @throws NoSuchElementException
+     *         if the specified base handler does not exist in this pipeline
+     * @throws NullPointerException
+     *         if the specified old handler or new handler is
+     *         {@code null}
+     */
+    ChannelPipeline addAfter(ChannelHandler baseHandler, ChannelHandler newHandler);
+
+    /**
+     * Inserts a {@link ChannelHandler} after the specified {@link ChannelHandler} of this pipeline.
+     *
+     * @param  baseHandler   the existing {@link ChannelHandler}
+     * @param  newName       the name of the {@link ChannelHandler} to insert after
```

Comment:
This language is wrong. It is "the name of the {@link ChannelHandler} to _be inserted_"